### DISCUSSION TITLE
Linq2db support

### DISCUSSION THREAD

**Discussion by jogibear9988** [Giorgi/EFCore.Visualizer#18] [General]
Maybe also linq2db orm could be supported?
Is the https://github.com/Giorgi/EFCore.Visualizer/blob/main/src/IQueryableObjectSource/EFCoreQueryableObjectSource.cs the only file directly using EFCore?

**Comment by Giorgi** [comment]
Yes, that is the only file specific to EF Core. Does linq2db have a way to get generated SQL from a query?

**Reply by jogibear9988** [reply]
You can get the LastQuery of the DataContext of the IQueryable:

<img width="1385" alt="image" src="https://github.com/Giorgi/EFCore.Visualizer/assets/364896/033541ab-487a-4005-be7c-94dd64c107a0">
